.yamato/config/universal__split.metafile

```diff
@@ -0,0 +1,46 @@
+project:
+  name: Universal_Split
+expression_trigger:
+  expression: ""
```

Comment:
Wouldn't it be simpler to have the expression trigger `expression: 'pull_request.target eq "<TARGET_BRANCH>" AND NOT pull_request.draft AND pull_request.changes.any match ["com.unity.render-pipelines.universal/**/*", "TestProjects/UniversalGraphicsTest/**/*"] AND NOT pull_request.push.changes.all match ["*.md", "**/*.md", "doc/**/*", "**/Documentation*/**/*"]'` here instead of duplicated in the 5 projects?